lxd/images.go

```diff
@@ -765,29 +600,13 @@ func imageDelete(d *Daemon, r *http.Request) Response {
 	if shared.PathExists(fmetaname) {
 		err = os.Remove(fmetaname)
 		if err != nil {
-			shared.Debugf("Error deleting image file %s: %s\n", fmetaname, err)
+			shared.Log.Error(
+				"Deleting image file", log.Ctx{"fmetaname": fmetaname, "err": err})
 		}
 	}
 
-	vgname, vgnameIsSet, err := getServerConfigValue(d, "core.lvm_vg_name")
-	if err != nil {
-		return InternalError(fmt.Errorf("Error checking server config: %v", err))
-	}
-
-	if vgnameIsSet {
-		err = shared.LVMRemoveLV(vgname, imgInfo.Fingerprint)
-		if err != nil {
-			return InternalError(fmt.Errorf("Failed to remove deleted image LV: %v", err))
-		}
+	if err = d.Storage.ImageDelete(imgInfo.Fingerprint); err != nil {
```

Comment:
Is it github messing up the diff or is there actually nothing inside this if statement besides its condition?